### ISSUE TITLE
WebSocket can't connect due to check_origin setting being absent

### ISSUE THREAD

**Issue by Yemolai** [Yemolai/yemolai-website#8]
Checking live logs we are getting the error:
```
[error] Could not check origin for Phoenix.Socket transport.
```
when loading the sign-in / register live views, these use WebSocket, looks like a setting `check_origin` is missing and should be defined in order to fix it.

**Comment by Yemolai** [comment]
Applied fix #7 , will check if that fixes the issue.

Based on investigation around the issue and [documentation](https://hexdocs.pm/phoenix/Phoenix.Endpoint.html#socket/3-common-configuration)

**Comment by Yemolai** [comment]
PR #7 fixes this issue
